src/macaron/slsa_analyzer/git_url.py

```diff
@@ -616,6 +619,20 @@ def parse_remote_url(
 
     # e.g., https://github.com/owner/project.git
     if parsed_url.scheme in ("http", "https", "ftp", "ftps", "git+https"):
+        # Handle Apache indirect GitHub links
+        if return_redirects:
+            redirect_list = defaults.get_list("repofinder", "redirect_urls", fallback=[])
+            if redirect_list and parsed_url.netloc in redirect_list:
+                response = send_get_http_raw(parsed_url.geturl(), allow_redirects=False)
+                if not response:
+                    return None
+                redirect_url = response.headers.get("location")
+                if not redirect_url:
+                    return None
+                # Pass the redirect_url through this function and prevent loops by setting the redirect parameter to an
+                # empty string.
```

Comment:
Sorry, I'm a bit confused about which empty string you are referring to here. Isn't the redirect parameter of type `bool`?

Reply:
I will update this comment to match reality.